.ci/cico_deploy.sh

```diff
@@ -37,19 +37,28 @@ echo "CHE VALIDATION: Verification skipped until job devtools-che-functional-tes
 # . cico_run_EE_tests.sh
 # echo "CHE VALIDATION: Verification passed. Pushing Che server image to prod registry."
 
+echo "CHE VALIDATION: Verification is disabled. Pushing Che server images to prod registry without verification."
+
 STAGE_IMAGE_TO_PROMOTE="${DOCKER_HUB_NAMESPACE}/che-server-multiuser:${CHE_SERVER_DOCKER_IMAGE_TAG}"
-PROD_IMAGE_DEVSHIFT="push.registry.devshift.net/che/che-multiuser:${CHE_SERVER_DOCKER_IMAGE_TAG}"
 
-echo "CHE VALIDATION: Pushing image ${PROD_IMAGE_DEVSHIFT} to devshift"
+DEVSHIFT_TAG=$(echo $GIT_COMMIT | cut -c1-${DEVSHIFT_TAG_LEN})
```

Comment:
What if `$GIT_COMMIT` is not set? I would rather test that the variable is set as we do for credentials.

Reply:
ok, will update PR - if `GIT_COMMIT` is not set script will abort